components/Ink/src/MDCInkView.h

```diff
@@ -125,6 +125,11 @@ typedef NS_ENUM(NSInteger, MDCInkStyle) {
  */
 - (void)cancelAllAnimationsAnimated:(BOOL)animated;
 
+/**
+ Add an Ink sublayer without any animation
+ */
+- (void)addInkSublayerWithoutAnimation;
```

Comment:
We spoke offline, please just update the current API to support non-animated Ink "start" and "end".

Reply:
done